src/index.js

```diff
@@ -80,12 +92,17 @@ export default class AutoCheckElement extends HTMLElement {
       this.input.dispatchEvent(new CustomEvent('autocheck:complete', {bubbles: true, cancelable: true}))
     }
 
+    if (this.required) {
+      this.input.setCustomValidity('Verifying...')
```

Comment:
This can use the ellipsis character <kbd>Option-;</kbd>: Verifying…

Reply:
TIL about <kbd>Option</kbd> + <kbd>;</kbd> 🙀